File: app/src/main/java/com/pens/it/d4b2018/clara_android/backup/data/models/LoginRequest.java
package com.pens.it.d4b2018.clara_android.backup.data.models;

import androidx.annotation.NonNull;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LoginRequest extends BaseModel {

    @SerializedName("email")
    @Expose
    private String email;

    @SerializedName("password")
    private String password;

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @NonNull
    @Override
    public String toString() {
        return "LoginRequest{" +
                "email='" + email + '\'' +
                ", password='" + password + '\'' +
                '}';
    }

    public LoginRequest(String email, String password) {
        this.email = email;
        this.password = password;
    }
}

File: app/src/main/java/com/pens/it/d4b2018/clara_android/mvvm/data/models/LoginRequest.kt
package com.pens.it.d4b2018.clara_android.mvvm.data.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LoginRequest(
        @Expose
        @SerializedName("email")
        var email: String,

        @SerializedName("password")
        var password: String
)